### PR TITLE
rm stripe/payment stuff

### LINKED ISSUE #16391: Remove Stripe integration
As part of cleaning up after the discontinuation of the Promoted Add-ons pilot, we should remove any Stripe / payment integration code we have on the server.

### PR DESCRIPTION
**Pull request by eviljeff**
fixes #16391

### CODE REVIEW

File: static/css/impala/developers.less [hunk 1]
```diff
@@ -2,25 +2,32 @@
 
 .developer-hub {
     #masthead .site-title {
-        small, strong {
+
+        small,
+        strong {
```
Comment (willdurand) [inline static/css/impala/developers.less:7]: Why do we have so much changes here?

**Review by willdurand — APPROVED** [review]
More red than green award, congratulations!

r+wc 😢

File: static/css/impala/developers.less (hunk 1 above)
Comment (eviljeff) [reply static/css/impala/developers.less:7]: looks like me changing the file triggered a less reformatting setting/extension in VSC.  I can revert all the formatting changes?

Comment (willdurand) [reply static/css/impala/developers.less:7]: > I can revert all the formatting changes?

yes please, thanks!

Comment (eviljeff) [reply static/css/impala/developers.less]: https://github.com/mozilla/addons-server/pull/16454/commits/e02affa0ab427cc83c8415c0e3fe6f2b91299845 now with even less 🟩

**Pull request merged**